nodeup/pkg/model/network.go

```diff
@@ -88,3 +97,27 @@ func (b *NetworkBuilder) addCNIBinAsset(c *fi.ModelBuilderContext, assetName str
 
 	return nil
 }
+
+func (b *NetworkBuilder) builddFlannelTxChecksumOffloadDisableService() *nodetasks.Service {
+	const serviceName = "flannel-tx-checksum-offload-disable.service"
+
+	manifest := &systemd.Manifest{}
+	manifest.Set("Unit", "Description", "Disable TX checksum offload on flannel.1")
```

Comment:
does canal also use the name `flannel.1` ?

Reply:
yes, I tested this with Canal

Reply:
Yeah we run `canal` and `flannel.1` is the interface name

Reply:
Not in my setup
```
admin@ip-10-63-80-157:~$ ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group
...
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc mq state UP group 
...
3: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state 
...
```
This change breaks my cluster. The service `flannel-tx-checksum-offload-disable` returns an error that prevents `kops-configuration` to finish its job.
```
admin@ip-10-63-80-157:~$ sudo /sbin/ethtool -K flannel.1 tx-checksum-ip-generic off
Cannot get device feature names: No such device
```
I'm running on AWS using canal
```
  networking:
    canal: {}
```

Reply:
How does this block `kops-configuration` from finishing the job? Worst case scenario it stays as the last job.

Reply:
Sorry. My bad. Looks like flannel pod wasn't started because the node wasn't able to join the cluster. As a result, the device wasn't created and kops-configuration service log was spammed by the messages from flannel-tx-checksum-offload-disable service. I had to manually fix the issue with etcd manager certificate first(my cert was already expired and rolling upgrade to the latest kops 1.17.0 failed).